src/components/search/search-box.js

```diff
@@ -41,6 +44,55 @@ class SearchBox extends React.Component {
     this.setState({ modalOpen: false });
   }
 
+  singleLinksFacet = ({
+    className, // eslint-disable-line
+    label, // eslint-disable-line
+    onRemove,
+    onSelect,
+    options,
+    values = []
+  }) => {
+    const value = values[0];
+    const siteFilter = options.filter(opt => opt.value === this.props.site)[0];
```

Comment:
@colleenmcginnis to further illustrate:

Swiftype provides a list of unique values stored for the field `site` this function filters to find the `site` prop we send the to Search component. In this case we're looking for the value `API`. Then we can use the data from Switype to create the "API" toggle.

Reply:
Ohhh thanks @katydecorah! 💡